app/models/katello/erratum.rb

```diff
@@ -16,6 +16,7 @@ class Erratum < Katello::Model
     has_many :bugzillas, :class_name => "Katello::ErratumBugzilla", :dependent => :destroy, :inverse_of => :erratum
     has_many :cves, :class_name => "Katello::ErratumCve", :dependent => :destroy, :inverse_of => :erratum
     has_many :packages, :class_name => "Katello::ErratumPackage", :dependent => :destroy, :inverse_of => :erratum
+    has_many :repository_errata, :class_name => "Katello::RepositoryErratum", :dependent => :destroy, :inverse_of => :erratum
```

Comment:
Turns out this is defined already by PulpDatabaseUnit and adding this here breaks stuff, so you can just remove it!